sdk/iot/core/doc/mqtt_state_machine.md

```diff
@@ -136,3 +138,120 @@ _Example:_
         }
     }
 ```
+
+### Retrying Operations
+
+Retrying operations requires understanding two aspects: error evaluation (did the operation fail, should the operation be retried) and retry timing (how long to delay before retrying the operation). The IoT Embedded Devices SDK is supplying optional APIs for error classification and retry timing.
+
+#### Error policy
+
+The SDK will not handle protocol-level (WebSocket, MQTT, TLS or TCP) errors. The application-developer is expected to classify and handle errors the following way:
+
+- Authentication errors should not be retried.

```

Comment:
Pardon my ignorance here...Should this have some kind of qualifier about "retrying with the same parameters/state/etc"
Or is that assumed?

Reply:
I will modify this to call out "operations" need to be retried. I can also change "should be considered retriable" to "may be retried".
The retry itself means different parameters will be used at least at some level in the stack so I don't want to call that out. State has already changed (operation failed).